samples/peripheral/802154_phy_test/modules/app_rpc/zephyr/Kconfig

```diff
@@ -0,0 +1,11 @@
+#
+# Copyright (c) 2021 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+config APP_RPC
```

Comment:
I see that this a general module, which could also be used in other applications, and that is also why this is `APP_RPC` and not `PTT_APP_RPC` - however in this case maybe we should consider moving it, for example to `nrf5340/remote_shell`.